data/mods/ssb/conditions.ts

```diff
@@ -63,6 +63,21 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
 			this.add(`c|${getName('Aelita')}|X.A.N.A. is finally finished for good.`);
 		},
 	},
+	aegii: {
+		noCopy: true,
+		onStart(pokemon) {
+			this.add(`c|${getName('aegii')}|${[`stream fiesta!!! https://youtu.be/eDEFolvLn0A`, `stream more&more!!! https://youtu.be/mH0_XpSHkZo`, `stream wannabe!!! https://youtu.be/fE2h3lGlOsk`, `stream love bomb!!! https://youtu.be/-SK6cvkK4c0`][this.random(4)]}`);
+			// Decide whether aegii will be special or not
+			if (!pokemon.m.hasMovesetFixed) return;
+			if (pokemon.set.moves[0] === "Shadow Ball") pokemon.m.hasMovesetFixed = true; // aegii has a special moveset.
```

Comment:
see previous comment, you wont need these after above

Reply:
I think this approach is better since all you have to do is just set a flag instead of runninga counter everytime